modules/openapi-generator-gradle-plugin/build.gradle

```diff
@@ -208,13 +210,17 @@ javadoc {
     }
 }
 
-tasks {
-    closeRepository {
-        onlyIf { nexusPublishing.useStaging.get() }
-    }
-    releaseRepository{
-        onlyIf { nexusPublishing.useStaging.get() }
-    }
-}
-
-publishToNexus.dependsOn 'check'
+//tasks {
+//    closeRepository {
+//        onlyIf {
+//            nexusPublishing.useStaging.any()
+//        }
+//    }
+//    releaseRepository{
+//        onlyIf {
+//            nexusPublishing.useStaging.any()
+//        }
+//    }
+//}
+
+//nexusPublishing.dependsOn 'check'
```

Comment:
May I know why these are commented out?

Reply:
Ah yes, good spot, basically `nexusPublishing` task no longer exists, I am not sure which one should replace it. Here is the list of available publishing tasks:

```
closeAndReleaseSonatypeStagingRepository - Closes and releases open staging repository in 'sonatype' Nexus instance.
closeAndReleaseStagingRepository - Closes and releases open staging repositories in the following Nexus instance: 'sonatype'
closeSonatypeStagingRepository - Closes open staging repository in 'sonatype' Nexus instance.
generateMetadataFileForMavenJavaPublication - Generates the Gradle metadata file for publication 'mavenJava'.
generateMetadataFileForOpenApiGeneratorPluginMarkerMavenPublication - Generates the Gradle metadata file for publication 'openApiGeneratorPluginMarkerMaven'.
generateMetadataFileForPluginMavenPublication - Generates the Gradle metadata file for publication 'pluginMaven'.
generatePomFileForMavenJavaPublication - Generates the Maven POM file for publication 'mavenJava'.
generatePomFileForOpenApiGeneratorPluginMarkerMavenPublication - Generates the Maven POM file for publication 'openApiGeneratorPluginMarkerMaven'.
generatePomFileForPluginMavenPublication - Generates the Maven POM file for publication 'pluginMaven'.
publish - Publishes all publications produced by this project.
publishAllPublicationsToSonatypeRepository - Publishes all Maven publications produced by this project to the sonatype repository.
publishMavenJavaPublicationToMavenLocal - Publishes Maven publication 'mavenJava' to the local Maven repository.
publishMavenJavaPublicationToSonatypeRepository - Publishes Maven publication 'mavenJava' to Maven repository 'sonatype'.
publishOpenApiGeneratorPluginMarkerMavenPublicationToMavenLocal - Publishes Maven publication 'openApiGeneratorPluginMarkerMaven' to the local Maven repository.
publishOpenApiGeneratorPluginMarkerMavenPublicationToSonatypeRepository - Publishes Maven publication 'openApiGeneratorPluginMarkerMaven' to Maven repository 'sonatype'.
publishPluginMavenPublicationToMavenLocal - Publishes Maven publication 'pluginMaven' to the local Maven repository.
publishPluginMavenPublicationToSonatypeRepository - Publishes Maven publication 'pluginMaven' to Maven repository 'sonatype'.
publishToMavenLocal - Publishes all Maven publications produced by this project to the local Maven cache.
publishToSonatype - Publishes all Maven publications produced by this project to the 'sonatype' Nexus repository.
releaseSonatypeStagingRepository - Releases closed staging repository in 'sonatype' Nexus instance.
```

Probably `publishToSonatype`, as description is most similar to `pubishToNexus`:

(old description) `publishToNexus - Publishes all Maven publications produced by this project to Nexus.`

Reply:
```
publishToSonatype - Publishes all Maven publications produced by this project to the 'sonatype' Nexus repository.
```

Yes, please use it as a replacement of `nexusPublishing`

Reply:
That's so strange, when I run `gradle tasks` it finds `publishToSonatype` but when I run it it crashes as task not found. `publish` does the same thing, so used it

Reply:
I'll try to test it over the weekend.